libs/common/src/test-helpers/test-api-error.ts

```diff
@@ -1,8 +1,18 @@
 import { HttpStatusCode } from 'axios';
 import { ApiError } from '../errors';
+import { ApiErrorCode } from '../types';
+
+type TestApiErrorParams = {
+  status?: HttpStatusCode;
+  message?: string;
+  code?: ApiErrorCode;
+  cause?: unknown;
+};
 
 export class TestApiError extends ApiError {
-  constructor(status?: number, message?: string) {
-    super({ status: status ?? HttpStatusCode.InternalServerError, message: message ?? '' });
+  constructor(testApiErrorParams: TestApiErrorParams = {}) {
+    const defaults = { status: HttpStatusCode.InternalServerError, message: '' };
+    const apiErrorConstructorParams = { ...defaults, ...testApiErrorParams };
```

Comment:
My only comment would be that we can pass explicitly undefined values through which would then replace the defaults (vs the coalescing operator above)

Reply:
I think this should be okay -- If we're explicitly overriding this now, it's because we want to explicitly override I'd think